alvr/session/src/settings.rs

```diff
@@ -881,7 +880,7 @@ pub struct ControllersConfig {
     #[schema(flag = "real-time")]
     pub tracked: bool,
 
-    #[schema(flag = "real-time")]
+    #[schema(flag = "steamvr-restart")]
```

Comment:
Does this actually need to be restart?

Reply:
Yeah. If a setting is steamvr-restart makes sense that all parent settings are also steamvr-restart. In this case, that setting controls if to initialize the controller skeleton input as Estimated or Partial. Not sure if it can be reinitialized at runtime, but we don't do that anyway.